README.md

```diff
@@ -1,8 +1,58 @@
 
 # transit-movements-message-parser
 
-This is a placeholder README.md for a new repository
+This is a PoC to determine how we can integrate with MDTP services for file transfer, namely Upscan, Object-Store and SDES. This is not intended for production code.
+
+## Running Locally
+
+You will need to ensure that you have the following services running:
+
+* upscan-stub
+* object-store (in stub mode)
+* secure-data-exchange-proxy
+* sdes-stub
+* internal-auth
+
+If using Service Manager, you can run the following (which will redirect the proxy call for the file ready notification to the sdes stub):
+
+```bash
+sm2 --start UPSCAN_STUB OBJECT_STORE_STUB SDES_PROXY SDES_STUBS INTERNAL_AUTH --appendArgs '{"SDES_PROXY": ["-Dauditing.enabled=false", "-Dmicroservice.services.acl.unrestricted-file-types.0=S18", "-Dmicroservice.services.event.port=9191", "-Dmicroservice.services.event.path=/sdes-stub"]}'
```

Comment:
Or sm --start.

Reply:
`sm` has been deprecated in favour of `sm2`.